packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_misc.spec.ts

```diff
@@ -1,43 +1,12 @@
+import _ from 'lodash'
+
 // @ts-ignore / session support is needed for visiting about:blank between tests
 context('multi-domain misc', { experimentalSessionSupport: true, experimentalMultiDomain: true }, () => {
   beforeEach(() => {
     cy.visit('/fixtures/multi-domain.html')
     cy.get('a[data-cy="dom-link"]').click()
   })
 
-  it('verifies number of cy commands', () => {
```

Comment:
Moved this test out of the context block since it doesn't need the navigation in the beforeEach. This test would also fail if it was run with any other tests that caused the `getAll` testing command to get added. Updated the test to remove the `getAll` command from the `actualCommands` if it was added.